Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -38,13 +38,10 @@ class Digitizer
   int mEventID = 0;
   int mSrcID = 0;
 
-  float mWion;   //  Ionization potential
```

Comment:
In which class will W_ion be considered in the future?

Reply:
It is only used in the ProcessHits method in the Detector class. Now hits contains the total charge instead of deposited energy, to avoid doing the conversion two times, and setting the ionization energy on several places too.